### ISSUE TITLE
"Bookmark" Section is Totally Missing

### ISSUE THREAD

**Issue by YunusDange** [arshadansari27/fitranginew#139]
I think we have missed to add this section although it is there in UI designs (PSD's Given by me)

1) It will come under My Profile as a Seperate Tab called "Bookmark" 
(right now we have tabs like Followers & Following which we can removed from it's present position becoz as per UI that list should come near Profile image tab created for the same purpose.
eg. see below:
![user-profile](https://cloud.githubusercontent.com/assets/7126709/7578483/48f1de94-f873-11e4-9648-cce0e3c6dbba.jpg)

Bookmark Section like below:
![user-profile-tabs-bookmarks](https://cloud.githubusercontent.com/assets/7126709/7578497/726cc19e-f873-11e4-83cd-38d8e97004f9.jpg)

2) Right now Bookmark section will have relation with 2 things 1is Journals & 2 nd is Discussion we should have "Add to Bookmark" Button on both this content type

3) In Future it will also relate to "Upcoming Trips" "Gears & Equipements" "Camp Sites" etc. people can add those in there bookmark list

**Comment by arshadansari27** [comment]
![screenshot](https://cloud.githubusercontent.com/assets/423442/7583976/afe65988-f8b7-11e4-9fd7-df7353ba2a99.png)


Bookmarks will be added.. but for now lets keep it for later... after we start marketing... fix the other stuff first..
as for followers and following it is there

**Comment by YunusDange** [comment]
Okkz
for followers and following
yes it is there....but we have planned earlier to show popup when user click on those icons and Followers & Followings can be shown in those popups rather than creating a separate TAB's  for it...so that we can include other TAB's for user profile.......Like "Bookmark" "Photos/Videos" & later "Event Calender" etc

**Comment by YunusDange** [comment]
![screenshot-2015-05-12-15 08 18](https://cloud.githubusercontent.com/assets/7126709/7584133/dddec7d4-f8b8-11e4-8d36-f7f202801473.jpg)

**Comment by arshadansari27** [comment]
@YunusDange ..

How about merging folllowers and following  in a single tab
and same can be done with bookmarks ...
journals/discussion/adventures can have a section on top for bookmarks and then the general will follow.. lemee know wat u think..
https://github.com/arshadansari27/fitranginew/issues/139#fullscreen

**Comment by arshadansari27** [comment]
I suggest moving this to 10th June Milestone either ways.. Coz it wont be completed on time before that...

**Comment by YunusDange** [comment]
@arshadansari27 

1) I have better option than merging folllowers and following Tabs
We already have a buttons beside profile indicating number of folllowers and following...eg. below
![afe65988-f8b7-11e4-9fd7-df7353ba2a99](https://cloud.githubusercontent.com/assets/7126709/7743442/7a73030c-ffb8-11e4-8140-d5268e860301.png)

 ...Let's make it click-able and once user click on any of those button a popup list should appear showing folllowers and following as per the click (this is the original idea in the UI i have created) this will completely remove the folllowers and followings from TAB's and we can adjust other TAB's in there place (like Bookmark)

2) journals/discussion/adventures can have a section on top for bookmarks and then the general will follow.. lemee know wat u think......NOT UNDERSTAND THIS????

What i have designed earlier as per UI is we should have a TAB name "Bookmarks" in Users Profile and it will content all the bookmarks "journals, discussion & adventures" divided in sections....that will be much better...rather than user going to different sections to find his bookmarks.

**Comment by arshadansari27** [comment]
I agree to the last point of keeping a separate tab for all kinds of bookmarks

Secondly, I do not agree to pop for followers and following.. I am trying to reduce pop ups... If possible, we would remove all pop ups.. they are short cut for less time.. but not really that responsive as you may have noticed.... so let's just add them in same tab .. that is my opinion

**Comment by YunusDange** [comment]
Got it...

So let's keep a single Tab .....keep "Followers" Tab & create "Following" section in that (no need to think on this much) keep it like zomato eg.
https://www.zomato.com/alltimefamished#network

they have "Followers" Tab and Included "Following" list in that only ...it make sense
